gateway/src/apicast/backend/cache_handler.lua

```diff
@@ -34,28 +34,29 @@ local function fetch_cached_key()
   return ok and stored
 end
 
-function _M.handlers.strict(cache, cached_key, response, ttl)
-  if response.status == 200 then
+function _M.handlers.strict(cache, cached_key, status_code, ttl)
+  if status_code == 200 then
     -- cached_key is set in post_action and it is in in authorize
     -- so to not write the cache twice lets write it just in authorize
 
     if fetch_cached_key(cached_key) ~= cached_key then
-      ngx.log(ngx.INFO, 'apicast cache write key: ', cached_key, ', ttl: ', ttl, ' sub: ')
+      ngx.log(ngx.INFO, 'apicast cache write key: ', cached_key,
+                        ', ttl: ', ttl, ' sub: ')
```

Comment:
Hm. The `sub: ` there does not really say anything. I guess 3e can remove it as it is some copy paste error.

Reply:
Ah your right. I didn't pay attention to this line and then copied it to the new policy.